lib/p5.js

```diff
@@ -5168,7 +5168,7 @@ amdclean['src_app'] = function (require, core, p5Color, p5Element, p5Graphics, p
   'use strict';
   var p5 = core;
   var _globalInit = function () {
-    if (!window.PHANTOMJS) {
+    if (!window.PHANTOMJS && !window.mocha) {
       if (window.setup && typeof window.setup === 'function' || window.draw && typeof window.draw === 'function') {
         new p5();
```

Comment:
mocha defines a global "setup" so a new ```p5``` instance was being created which called mocha's ```setup()``` method which broke mocha in the browser.